### ISSUE TITLE
Enable Track memory changes

### ISSUE THREAD

**Issue by hamann** [coreos/bugs#2048]
# Issue Report #

### Environment ###

All

### Desired Feature ###

Please enable `Track memory changes (CONFIG_MEM_SOFT_DIRTY)`, which is necessary to make incremental dumps with [criu](https://criu.org/Installation)

**Comment by lucab** [comment]
I was a bit concerned with the performance impact of this, so for reference I found the following statement in the [initial patchset description](https://lwn.net/Articles/546184/):
```
As long as soft dirty bits are not cleared, no performance
penalty occur, since the soft dirty bit and the regular dirty bit are set at 
the same time within the same instruction. When soft dirty is cleared via 
clear_refs, the task in question might slow down, but it will depend on how
actively it uses the memory.
```

**Comment by bgilbert** [comment]
This is done in https://github.com/coreos/coreos-overlay/pull/2663, which should be included in 1492.0.0.
